pkg/bench/tpcc/tpcc_bench_test.go

```diff
@@ -80,17 +80,26 @@ func BenchmarkTPCC(b *testing.B) {
 		}),
 	}
 
-	for _, opts := range []options{
-		{workloadFlag("mix", "newOrder=1")},
-		{workloadFlag("mix", "payment=1")},
-		{workloadFlag("mix", "orderStatus=1")},
-		{workloadFlag("mix", "delivery=1")},
-		{workloadFlag("mix", "stockLevel=1")},
-		{workloadFlag("mix", "newOrder=10,payment=10,orderStatus=1,delivery=1,stockLevel=1")},
-	} {
-		b.Run(opts.String(), func(b *testing.B) {
-			newBenchmark(append(opts, baseOptions...)).run(b)
+	for _, impl := range []string{"literal", "optimized"} {
+		b.Run(impl, func(b *testing.B) {
+			implOpts := baseOptions[:]
+			if impl == "literal" {
+				implOpts = append(implOpts, workloadFlag("literal-implementation", "true"))
+			}
+			for _, opts := range []options{
+				{workloadFlag("mix", "newOrder=1")},
+				{workloadFlag("mix", "payment=1")},
+				{workloadFlag("mix", "orderStatus=1")},
+				{workloadFlag("mix", "delivery=1")},
+				{workloadFlag("mix", "stockLevel=1")},
+				{workloadFlag("mix", "newOrder=10,payment=10,orderStatus=1,delivery=1,stockLevel=1")},
```

Comment:
could add a comment here that this is the official "tpcc spec mix".

Reply:
also - I'll leave it to you to decide but want to suggest: could make sense to give this a shorter name ("default").

Reply:
I'll add a commit in the follow-up PR to make all these names better.